pipeline_dp/pipeline_backend.py

```diff
@@ -172,6 +172,46 @@ def flatten(self, cols: Iterable, stage_name: str):
     def distinct(self, col, stage_name: str):
         """Returns a collection containing distinct elements of the input collection."""
 
+    def size(self, col, stage_name: str):
```

Comment:
Done. But I named the file more specific: "pipeline_composite_functions", there is a best practice to avoid names utils, because they are not specific enough and it will become a dump of different methods not related to each other. WDYT?

Reply:
Done